packages/flutter/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: A framework for writing Flutter applications
 homepage: https://flutter.dev
 
 environment:
-  sdk: ">=2.12.0-0 <3.0.0"
+  sdk: ">=2.17.0-0 <3.0.0"
```

Comment:
Was this bump in the min version intentional?

Reply:
No, fixed!